code/game/objects/items/weapons/extinguisher.dm

```diff
@@ -151,9 +151,6 @@
 						if(isliving(atm)) //For extinguishing mobs on fire
 							var/mob/living/M = atm
 							M.ExtinguishMob()

```

Comment:
I left that in since water removes firestacks on reaction_mob but doesn't actually extinguish. I just assumed the extinguisher supposed to be better than spraying people with water.